oceanprotocol/ocean-node
# Making the dashboard optional

Fixes #327 

Changes proposed in this PR:

- Making the dashboard optional
- Dashboard still runs by default, the new env is optional

To test:
```
export DASHBOARD="false"
npm run build
npm run start 
```

Navigate to http://localhost:8000/dashboard/ and there should be no dashboard. 

Now run: 
```
export DASHBOARD="true"
npm run build
npm run start 
```

and the dashboard should be running at http://localhost:8000/dashboard/
